CHANGELOG.md

```diff
@@ -7,6 +7,170 @@ and this project adheres to [Semantic Versioning](https://book.async.rs/overview
 
 ## [Unreleased]
 
+## [2.0.0] - 2020-10-03
+
+[Docs](https://docs.rs/surf/2.0.0)
+
+This major release of Surf contains substantial improvements through a variety of changes and additions.
+
+_(Note: this is a cumulative list of changes since Surf 1.0.3)_
+
+Notable mentions include:
+- Uses stable standard library [`futures`][]!
+- Much more type compatibility with [Tide][] via [http-types][]!
+- Re-usable `Client` which is able to make use of connection pooling [under the hood](https://github.com/http-rs/http-client).
+- Reduced generics for `Client` and `Middleware`.
+- Re-worked `Middleware` to use [`async_trait`][].
+
+### Major Changes
+Surf 2 contains some large API changes, as noted here:
+
+#### http-types
+
+Surf has switched the common backing type interface (`surf::http`) from the [http][] crate to [http-types][].
+
+This affects any type that came from `surf::http`, such as `StatusCode` ([old][StatusCode http]|[new][StatusCode http-types]), and includes some new patterns, such as [`Body`][].
+
+#### Errors
+
+`surf::Exception`, which was a plain `Box<dyn Error + Send + Sync + 'static>`, is no more.
+
+Surf now exports a structured [`surf::Error`][] type, which holds a `StatusCode` alongside a dynamic error object.
+
+#### Middleware
+
+**Old middleware:**
+```rust
+use futures::future::BoxFuture;
+use surf::middleware::{Next, Middleware, Request, Response, HttpClient};
+
+impl<C: HttpClient> Middleware<C> for Logger {
+    fn handle<'a>(
+        &'a self,
+        req: surf::Request,
+        client: C,
+        next: Next<'a, C>,
+    ) -> BoxFuture<'a, Result<Response, surf::Exception>> {
+        Box::pin(async move {
+            Ok(res)
+        })
+    }
+}
+```
+
+**New middleware:**
+```rust
+use surf::middleware::{Middleware, Next};
+use surf::{Client, Request, Response, Result};
+
+#[surf::utils::async_trait]
+impl Middleware for Logger {
+    async fn handle(
+        &self,
+        req: Request,
+        client: Client,
+        next: Next<'_>,
+    ) -> Result<Response> {
+        Ok(res)
+    }
+}
+```
+
+#### RequestBuilder
+
+The top-level convineience request methods, `surf::get()`, `surf::post()`, etc, now return a [`RequestBuilder`][] rather than a [`Request`][] directly.
```

Comment:
```suggestion
The top-level convenience request methods, `surf::get()`, `surf::post()`, etc, now return a [`RequestBuilder`][] rather than a [`Request`][] directly.
```